.travis.yml

```diff
@@ -13,19 +14,10 @@ matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
+  exclude:
+    - rvm: 2.4.5
+    - env: BRANCH=master
   include:
-    - rvm: 2.2.10
```

Comment:
Any reason to remove those?

Reply:
Because 2.2.10 and 2.3.8 are not in `rvm:`.

Reply:
Right, they are not in the main matrix and this is why we are adding in the `include:` matrix.

Reply:
Ah... `include`... I was thinking of `exclude`

Reply:
Fixed. I am not sure why build on Travis CI is failing. It is passing on https://travis-ci.org/eitoball/activeresource/builds/508732442